### PR TITLE
Rename fields in output validity proof

### PR DESCRIPTION
**Pull request by gligneul**
For more information see https://github.com/cartesi/rollups/issues/10

### CODE REVIEW

File: server-manager.proto [hunk 1]
```diff
@@ -237,13 +237,13 @@ message InspectStateResponse {
 
 // Validity proof for an output
 message OutputValidityProof {
-    uint64 input_index = 1; // Local input index within the context of the related epoch
-    uint64 output_index = 2; // Output index within the context of the input that produced it
+    uint64 input_index_within_epoch = 1; // Local input index within the context of the related epoch
+    uint64 output_index_within_epoch = 2; // Output index within the context of the input that produced it
```
[the remark this answers is not in the document]
Comment (gligneul) [reply server-manager.proto:241]: Makes sense!

**Review by guidanoli — APPROVED** [review]

**Review by vfusco — APPROVED** [review]

**Review by diegonehab — COMMENTED** [review]
Agree with changes so far, but I think we still have a ways to go.

We have the following fields:

```
StartSessionRequest.processed_input_count
InputMetadata.input_index
AdvanceStateRequest.current_input_index
InspectStateResponse.processed_input_count
OutputValidityProof.input_index_within_epoch
Proof.input_index
FinishEpochRequest.processed_input_count_within_epoch
```

They all refer to similar some form of index of the input, either since genesis or within epoch. But sometimes they are count, some times index, sometimes processed, sometimes current, sometimes neither. This should be simplified as much as possible. Perhaps we need two different field names at most?

Also, `OutputEnum` should be OutputType. The fact it is an enum is irrelevant.

**Comment by gligneul** [comment]
@diegonehab I agree with those changes, but I think they are out of scope for this PR and issue. My issue only covers the names in the output validity proof structure. This structure is particularly important because it is replicated in the onchain API, so we must modify it before the main-net release.

**Comment by diegonehab** [comment]
Will do those on a different PR.

**Pull request merged**